addons/http_routing/models/ir_http.py

```diff
@@ -302,11 +302,11 @@ def get_nearest_lang(cls, lang_code):
         """ Try to find a similar lang. Eg: fr_BE and fr_FR
             :param lang_code: the lang `code` (en_US)
         """
+        if not lang_code or lang_code in request.env['ir.module.module']._installed().keys():
```

Comment:
`.keys()` is redundant here.
But really, I'm not sure about this part. It may cause more bugs than it solves.

Once we have the second part of the fix, the issue will be a corner case that only affects developers with Croatian installed + HR module, and only while they are working in debug=assets mode. But if we change this it may break language routing as soon as there is a collision between language code and installed modules, which *may in fact happen*. And it will affect all users, not just developers working in a very specific setup.

And this is if we ignore the impact of the extra call. The routing stack is perf sensitive, and even if `_installed()` is cached, it's an extra hit.

Reply:
Ok I got your point, https://github.com/odoo/odoo/commit/269aa594111a152ad4b7714856ea745bfef57155 definitely 'revealed' the bug in the sense it did not kept it restricted to only when the `hr_HR` lang was installed.

I adapted this PR to restore that behavior so the bug won't appear unless `hr_HR` is installed.
Actually, the mentioned commit should've been written with `get_installed()` and not `get_available()` anyway, error during dev while changing methods to use the orm_cached ones.